### DISCUSSION TITLE
Use of two PCs, one for Plot and one for cultivation

### DISCUSSION THREAD

**Discussion by Edison5151** [Chia-Network/chia-blockchain#5619] [Support]
I have two machines, one to make the lots and the other to grow them, I would like to know how I transfer the lots created from the main to the second. Can I just move the hd to the second?

**Comment by aa2001** [comment]
If you have configured the machines, you can transfer both files and entire disks.

**Reply by Edison5151** [reply]
Okay, both of us are using the same wallet. If I connect the hd with the lots on the second machine and add the directory through the chia app they are already cultivating?

**Reply by aa2001** [reply]
Yes, when adding a path, Ghia will update and check the ptot list.

**Comment by BrandtH22** [comment]
@Edison5151 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
